tests/api_service_test.py

```diff
@@ -0,0 +1,16 @@
+from vcr_unittest import VCRTestCase
+from api_service import ApiService
+
+class TestApiService(VCRTestCase):
+
+    def test_create_desk_is_successful(self):
+        response = ApiService.create_desk(name = 'test desk')
+        json_response = response.json()
+
+        self.assertEqual(response.status_code, 201)
+        assert 'id' in json_response
+        self.assertEqual('test desk', json_response['name'])
+
+    def test_update_desk_occupied_status_is_successful(self):
+        response = ApiService.update_desk_occupied_status(id=1, occupied_status = True)
```

Comment:
These arguments are positional arguments and don't require being identified by name. Are you doing it for readability? Works for me.

Either, can we be consistent with spacing on either side of '=' sign.

Reply:
Yeah I did it mostly for readability. I find that for methods that take multiple arguments, being very explicit makes the code a lot more readable (also valid in Ruby land, i.e keyword arguments)

Reply:
Works for me, what about the styling? ie. spacing around `=`